dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2096,6 +2105,224 @@
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltResult"
+        }
+      ]
+    },
+    "ConversationSentimentResult": {
+      "description": "The result from sentiment analysis operation for each conversation item.",
+      "required": [
+        "conversationItems"
+      ],
+      "properties": {
+        "conversationItems": {
+          "description": "Enumeration of Sentiment operation results for all the conversation items in a conversation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationSentimentItemResult"
+          }
+        }
+      }
+    },        
+    "ConversationSentimentItemResult": {
+      "type": "object",
+      "required": [
+        "id",
+        "participantId",
+        "sentiment",
+        "confidenceScores",
+        "sentences"
+      ],
+      "properties": {
+        "id": {
+          "description": "The identifier for the conversation item",
+          "type": "string"
+        },
+        "participantId": {
+          "description": "The identifier for the speaker",
+          "type": "string"
+        },
+        "sentiment": {
+          "$ref": "#/definitions/Sentiment"
+        },                
+        "confidenceScores": {
+          "type": "object",
+          "$ref": "#/definitions/SentimentConfidenceScores"
+        },
+        "sentences": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SentenceSentiment"
+          }
+        }
+      }
+    },
+    "Sentiment": {
+      "type": "string",
+      "description": "Predicted sentiment.",
+      "enum": [
+        "positive",
+        "neutral",
+        "negative",
+        "mixed"
+      ],
+      "x-ms-enum": {
+        "name": "TextSentiment",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "positive",
+            "description": "Positive sentiment."
+          },
+          {
+            "value": "neutral",
+            "description": "Neutral sentiment."
+          },
+          {
+            "value": "negative",
+            "description": "Negative sentiment."
+          },
+          {
+            "value": "mixed",
+            "description": "Mixed sentiment."
+          }                   
+        ]
+      }
+    },
+    "SentenceSentiment": {
+      "type": "object",
+      "required": [
+        "text",
+        "sentiment",
+        "confidenceScores",
+        "offset",
+        "length"
+      ],
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The sentence text."
+        },
+        "sentiment": {
+          "$ref": "#/definitions/Sentiment"
+        },
+        "confidenceScores": {
+          "description": "The sentiment confidence score between 0 and 1 for the sentence for all classes.",
+          "$ref": "#/definitions/SentimentConfidenceScores"
```

Comment:
This type has a score for positive, negative, and neutral sentiment, but SentenceSentiment returns Sentiment which encompasses positive, negative, neutral, _and_ mixed. Is there not a score for mixed, or should the SentenceSentiment not include mixed? I believe in Text Analytics there is no "mixed" sentiment at the sentence level, only the document level.

Reply:
you are correct here. And based on the discussion, we've decided to remove the Sentence level Sentiment for this release